modules/tinymce/src/core/test/ts/browser/FormatterRemoveTest.ts

```diff
@@ -603,4 +604,14 @@ describe('browser.tinymce.core.FormatterRemoveTest', () => {
     editor.formatter.remove('blockquote');
     TinyAssertions.assertContent(editor, '<p>test test</p>');
   });
+
+  it('TINY-8755: Non-internal attributes are no longer removed', () => {
```

Comment:
```suggestion
  it('TINY-8755: Non-internal attributes are not removed', () => {
```
We generally want these to be using past tense since that's how they'll be read in future.